qa/rpc-tests/test_framework/util.py

```diff
@@ -563,7 +563,7 @@ def assert_equal(expected, actual, message=""):
     if expected != actual:
         if message:
             message = "; %s" % message 
-        raise AssertionError("(left == right)%s\n  left: <%s>\n right: <%s>" % (message, str(expected), str(actual)))
+        raise AssertionError("(expected == actual)%s\n expected: <%s>\n   actual: <%s>" % (message, str(expected), str(actual)))
```

Comment:
Even though the arguments are called `expected` and `actual`, it's not guaranteed that callers will actually use them that way. If a caller swaps around their meanings, this assertion message could become very confusing. I would revert this change.

Reply:
Meh, okay. I made this change specifically because I found `left` and `right` confusing in the resulting error messages. I would rather standardize on the order of arguments, but I guess that doing so requires checking every call site, and I'm not going to make those changes here.